logbook/oplog/log.go

```diff
@@ -368,6 +431,7 @@ type Log struct {
 	authorID string // authorID value cache. not persisted
 	parent   *Log   // parent link
 
+	ParentID  string // init id of parent, if one exists
```

Comment:
Does "one" refer to initID or parent? If the latter, could you change this to "init id of parent, if a parent exists"

Reply:
I'm realizing this isn't always the case it's more of a field that _may_ hold the initID of the parent log if one exists, but that really isn't being strictly enforced at this point. I'm going to change this to just `init id of parent Log`